hivemind/client/averaging/training.py

```diff
@@ -10,6 +10,7 @@
 from hivemind.utils import nested_flatten, nested_pack, get_logger, run_in_background
 
 logger = get_logger(__name__)
+LRSchedulerBase = getattr(torch.optim.lr_scheduler, '_LRScheduler', None)
```

Comment:
It should be possible to use _LRScheduler directly:
```
>>> import torch
>>> torch.optim.lr_scheduler._LRScheduler
<class 'torch.optim.lr_scheduler._LRScheduler'>
```
I'm not even sure that we need to define an alias for this class, since readers familiar with PyTorch code will recognize it